Backend/src/routes/stock_routes.js

```diff
@@ -1,9 +1,13 @@
 import Router from 'express';
 import bodyParser from 'body-parser';
-import { getStockPrice } from "../controller/stock-price.js";
+import { getStockPrice, getStockPrices } from "../controller/stock-price.js";
 
 export const stockRouter = new Router();
 
 stockRouter.get('/price', (req,res) => {
 	getStockPrice(req, res);
```

Comment:
I think the stock_routes file should pass the stock/stocks directly to the controller. Controller should not have to worry about req or res.